clients/privacy-center/tsconfig.json

```diff
@@ -13,7 +13,11 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "baseUrl": ".",
+    "paths": {
+      "~/*": ["./*"]
```

Comment:
Is there a preferred approach to not use an alias for relative paths?

Reply:
Sorry, not which question you're asking:
- "Do we prefer absolute paths over relative paths?"
    - Ctl code does but it hasn't been fully migrated over.
    - Generally okay with siblings and shallow children, just no `../`
- "Do we prefer `~/` over an alias for each top-level dir, e.g. `config/*: ["./config/*"]`
    - Ctl does. It looks like the admin-ui now has ops's alias because ops wasn't using `~/*` alias.
    - It's a bit annoying to add an alias whenever a top-level directory is added. (Inculding jest config)
    - Those aliases can also be confused for package names (`~` can't).

Reply:
I find using a path alias easier to manage than relative paths given that someone forgets to update the given relative path appropriately.